pkg/reconciler/accessor/istio/destinationrule_test.go

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package istio
+
+import (
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	istiov1alpha3 "istio.io/api/networking/v1alpha3"
+	"istio.io/client-go/pkg/apis/networking/v1alpha3"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	istioclientset "knative.dev/net-istio/pkg/client/istio/clientset/versioned"
+	fakeistioclient "knative.dev/net-istio/pkg/client/istio/injection/client/fake"
+	fakedrinformer "knative.dev/net-istio/pkg/client/istio/injection/informers/networking/v1alpha3/destinationrule/fake"
+	istiolisters "knative.dev/net-istio/pkg/client/istio/listers/networking/v1alpha3"
+
+	. "knative.dev/pkg/reconciler/testing"
+)
+
+var (
+	originDR = &v1alpha3.DestinationRule{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:            "dr",
+			Namespace:       "default",
+			OwnerReferences: []metav1.OwnerReference{ownerRef},
+		},
+		Spec: istiov1alpha3.DestinationRule{
+			Host: "origin.example.com",
+		},
+	}
+
+	desiredDR = &v1alpha3.DestinationRule{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:            "dr",
+			Namespace:       "default",
+			OwnerReferences: []metav1.OwnerReference{ownerRef},
+		},
+		Spec: istiov1alpha3.DestinationRule{
+			Host: "desired.example.com",
+		},
+	}
+)
+
+type FakeDestinatioRuleAccessor struct {
+	client   istioclientset.Interface
+	drLister istiolisters.DestinationRuleLister
+}
+
+func (f *FakeDestinatioRuleAccessor) GetIstioClient() istioclientset.Interface {
+	return f.client
+}
+
+func (f *FakeDestinatioRuleAccessor) GetDestinationRuleLister() istiolisters.DestinationRuleLister {
+	return f.drLister
+}
+
+func TestReconcileDestinationRule_Create(t *testing.T) {
```

Comment:
```suggestion
func TestReconcileDestinationRuleCreate(t *testing.T) {
```

Reply:
Has been sent (and merged) as a prereq PR already.